src/autocomplete.js

```diff
@@ -72,19 +72,27 @@ export default class Autocomplete extends Component {
   constructor (props) {
     super(props)
 
+    const setCorrectOptions = () => {
+      if (props.value) {
+        return [props.value];
+      } else if (props.defaultValue) {
+        return [props.defaultValue];
+      } else {
+        return [];
+      }
+    };
+
     this.state = {
       focused: null,
       hovered: null,
       clicked: null,
       menuOpen: false,
-      options:
-        (props.value && [props.value]) || props.defaultValue
-          ? [props.defaultValue]
-          : [],
+      options: setCorrectOptions(),
       query: props.value || props.defaultValue || '',
       selected: null,
       // Because in React is forbidden to change component's prop inside the component, we need to save the prop inside the state and change it later
-      showAllValuesOnFocus: props.showAllValues
+      showAllValuesOnFocus: props.showAllValues,
+      hideMenuAfterOptionClick: false,
```

Comment:
Cisto z citatelnosti, `hideMenuAfterOptionClick` nie je dobry nazov state. Evokuje to nieco co sa dynamicky nemeni v ramci vnutra komponentu. Proste bud sa ma menu skryt po kliknuti na option, alebo nema.

Volil by som skor nieco take, ze `optionWasClicked` alebo tak, co povies?

Reply:
premenujem